### PR TITLE
Dockerize 

### PR DESCRIPTION
**Pull request by bikerwhocodes**
Please create an organization on hub.docker and add me as a member so that I can push the docker file under your organization (currently the remote image is under mine i.e. bikerwhocodes).

### CODE REVIEW

File: README.md
Comment (antigravities) [inline README.md]: Can you please remove the single backticks here and in the other code blocks? When rendered they show literally, which I don't think was intended.

File: README.md
Comment (antigravities) [inline README.md]: This probably should be under its own heading? I'll leave it to you on this one, but "or" and "for headless mode" should probably be outside the code block.

File: README.md
Comment (antigravities) [inline README.md]: I do plan on publishing to Docker Hub immediately after I merge this using GitHub Actions, so this can probably be safely changed to `antigravities/levelup`.

(Ohp, missed your opening comment, sorry - this will be published to my personal account)

**Review by antigravities — CHANGES_REQUESTED** [review]
Thanks again for your contribution! I will presume everything is OK with you [Developer Certificate of Origin](https://developercertificate.org)-wise, so I'll add the Signed-off-by header in the final merge commit. Thanks also for writing documentation - if you could fix these small things, I'd really appreciate it:

**Comment by bikerwhocodes** [comment]
changes updated

**Review by antigravities — APPROVED** [review]
Awesome! Thanks so much!

**Pull request merged**
